.changeset/brave-pumas-deny.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/ui-components': minor
```

Comment:
I am not fully sure, but maybe it better fits to patch?
Like it is with other similar PRs - https://github.com/SAP/open-ux-tools/pull/879/files#diff-e5d3f21e663e649426773011139a16949b1247ba0335a4e28ad8246d2fc06128R2

Reply:
Thanks for pointing that out I'll change it now :)